content/file/059d6b2c-758b-4eec-86ac-8d0c6142bfd7.json

```diff
@@ -0,0 +1,87 @@
+{
+    "_links": {
+        "self": {
+            "href": "http:\/\/docroot.cardinalsites.loc\/sites\/default\/files\/media\/image\/banner-110304-0827.jpg"
```

Comment:
This path is not going to be the same for everyone and will cause change on every import/export

Reply:
yeah. i forgot to replace it with `default` If we just find and replace urls after export it'll help with that. i dont know of any other way.